pkg/scheduler/scheduler.go

```diff
@@ -52,6 +53,9 @@ const (
 	SchedulerError = "SchedulerError"
 )
 
+// ErrSkipScheduleBYEquivalenceClass is used to describe the error that skipping schedule pod because of EquivalenceClass.
+var ErrSkipScheduleBYEquivalenceClass = fmt.Errorf("skip schedule pod because EquivalenceClass")
```

Comment:
Try to make the message more clear, how about: 

`skip as its EquivalenceClass pod failed to schedule`

Reply:
Yeah thank you~ Done